pkg/controllers/disruption/consolidation_test.go

```diff
@@ -157,7 +157,7 @@ var _ = Describe("Consolidation", func() {
 			fakeClock.Step(10 * time.Minute)
 			ExpectSingletonReconciled(ctx, disruptionController)
 			ExpectMetricGaugeValue(disruption.EligibleNodes, 0, map[string]string{
-				metrics.ReasonLabel: string(v1.DisruptionReasonUnderutilized),
+				metrics.ReasonLabel: "underutilized",
```

Comment:
why not just do ToLower? Why take it away from the variable?

Reply:
yeah was wondering similar thing :)

Reply:
Because I want to test the actual static value representation matches my expectation. Imagine I have a typo in the constant